### DISCUSSION TITLE
Picking which data to replicate.

### DISCUSSION THREAD

**Discussion by oranichu** [backube/volsync#108] [Q&A] [answered]
Hey all, 
First off - amazing work! loved reading and watching the youtube explanation videos, very detailed and informative.
I got a question regarding whether VolSync can differ which data to replicate over to a different cluster over some desired category. and if so how can such a thing be achieved :)? 

Thanks!

**Comment by JohnStrunk** [comment, marked as answer]
Right now, data is replicated at the granularity of a PVC: the ReplicationSource/ReplicationDestination set up replication for the contents of a single PVC. If your application has some data that should be replicated and some that shouldn't, the only method to do that today would be to use separate PVCs.

It may be possible to enhance the RS/RD to take include/exclude patterns (e.g., only replicate the `*.zip` files in this PVC), but since the replication is offloaded to the various movers, they would all need to be modified to support this capability. I believe the existing & planned movers all support patterns, but we could certainly end up with ones that don't.
